index.html

```diff
@@ -0,0 +1,204 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+	<meta charset="UTF-8" />
+	<meta name="viewport" content="width=device-width, initial-scale=1.0" />
+	<meta http-equiv="X-UA-Compatible" content="ie=edge" />
+	<title>Finish_Mobile_Version</title>
+	<link rel="preconnect" href="https://fonts.googleapis.com" crossorigin />
+	<script src="https://kit.fontawesome.com/fe8431d094.js" crossorigin="anonymous"></script>
+	<link rel="preconnect" href="https://fonts.gstatic.com" crossorigin />
+	<link href="https://fonts.googleapis.com/css2?family=Poppins&display=swap" rel="stylesheet" />
+	<link rel="stylesheet" href="style.css" />
+</head>
+
+<body>
+
+	<header>
+		<nav class="buttons">
+			<h2 class="logo"><a href="#">My Logo</a></h2>
+			<a href="#" class="menu-icon"><i class="fa-solid fa-bars"></i></a>
+		</nav>
+	</header>
+
+	<section class="hero">
+		<div class="aboutme">
+			<h1>I'm Fentahun.<br>Glad to see<br>you!</h1>
+			<p>
+				I'm a software developer! I can help you<br />
+				build a product, feature or website look<br />
+				through some of my work and<br />
+				experience! if you like what you see and<br />
+				have a project you need coded, don't<br />
+				hesitate to contact me.
+			</p>
+			<a href="#" class="connect-btn">LET'S CONNECT</a>
+			<ul class="socials">
+				<li><a href="#"><img src="./images/Twitter.jpg" alt="twitter icon"></a></li>
+				<li><a href="#"><img src="./images/Linkedin icon.jpg" alt="Linkedin icon"></a></li>
+				<li><a href="#"><img src="./images/M icon.jpg" alt="Medium icon"></a></li>
+				<li><a href="#"><img src="./images/Github.jpg" alt="Github icon"></a></li>
+				<li><a href="#"><img src="./images/Peace.jpg" alt="Peace icon"></a></li>
+			</ul>
+		</div>
+	</section>
+
+	<!-- project -->
+	<section class="card">
+		<section class="page-2">
+			<div class="wrapper">
+				<div class="img">
+					<img src="./images/Page 2 image.png" alt="page 2 image">
+				</div>
+				<div class="text">
+					<h1>Tonic</h1>
+					<ul>
+						<h5>CANOPY</h5>
+						<li>Back End Dev</li>
+						<li>2015</li>
+					</ul>
+					<p>A daily selection of privately<br>personalized reads; no accounts or<br>sign-ups required.
+					</p>
+				</div>
+				<ul class="btn">
+					<li><a href="#" class="html">HTML</a></li>
+					<li><a href="#" class="css">CSS</a></li>
+					<li><a href="#" class="j555avascript">JavaScript</a></li>
+				</ul>
+				<div class="container">
+					<a href="#">See project</a>
+				</div>

```

Comment:
- [ ] Your button needs to have a margin-bottom, I kindly suggest you fix it 👍🏿 
- [ ] Your Design 
![image](https://github.com/Fentahunn/finish_mobile_version/assets/68823331/3f9b38b1-0a5b-4d16-a88b-7572f6698266)

- [ ] Figma Design
![image](https://github.com/Fentahunn/finish_mobile_version/assets/68823331/3e3875fa-4992-4074-b223-d3830ff0266a)

- [ ] Great job on implementing the button! However, in order to align with the design requirements, please make the following updates:

- Apply the necessary CSS classes or styles to the button element to achieve the active, hover, and pressed button styles as specified in the design.

- [ ] Here is the required button style for each event

![image](https://github.com/Fentahunn/finish_mobile_version/assets/68823331/c8336ade-f036-41e3-b50c-538497d4f6e8)